### PR TITLE
V2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: README.md [hunk 1]
```diff
@@ -25,28 +26,6 @@ Element is a new type of messaging app. You choose where your messages are store
...
-We do not recommend running Element from the same domain name as your Matrix
-homeserver (Synapse).  The reason is the risk of XSS (cross-site-scripting)
-vulnerabilities that could occur if someone caused Element to load and render
-malicious user generated content from a Matrix API which then had trusted
-access to Element (or other apps) due to sharing the same domain.
-
-We have put some coarse mitigations into place to try to protect against this
-situation, but it's still not good practice to do it in the first place.  See
-https://github.com/vector-im/riot-web/issues/1977 for more details.
-
```
Comment (Josue-T) [inline README.md:49]: Well, why do you remove this as I think it still important.

File: scripts/upgrade [hunk 2]
```diff
@@ -12,68 +12,33 @@ source /usr/share/yunohost/helpers
...
-app=$YNH_APP_INSTANCE_NAME
-
-domain=$(ynh_app_setting_get --app=$app --key=domain)
-path_url=$(ynh_app_setting_get --app=$app --key=path)
-final_path=$(ynh_app_setting_get --app=$app --key=final_path)
-default_home_server=$(ynh_app_setting_get --app=$app --key=default_home_server)
-
-login_for_welcome=$(ynh_app_setting_get --app=$app --key=login_for_welcome)
+#default_home_server=$(ynh_app_setting_get --app=$app --key=default_home_server)
+#login_for_welcome=$(ynh_app_setting_get --app=$app --key=login_for_welcome)
```
Comment (Josue-T) [inline scripts/upgrade:18]: Can you clarify this ?

**Comment by ericgaspar** [comment]
This branch as been reworked in https://github.com/YunoHost-Apps/element_ynh/tree/v2 and is still work in progress. the testing branch is open to review.

**Pull request merged**